Lib/WebHandler.py

```diff
@@ -227,11 +227,47 @@ def getProperties(self):
   def isRegisteredUser(self):
     return self.__credDict.get('validDN', "") and self.__credDict.get('validGroup', "")
 
-  def getSessionData(self):
-    return self.__sessionData.getData()
+  def getSessionData(self, opt=None):
+    return self.__sessionData.getData(opt)
 
   def getAppSettings(self, app=None):
-    return Conf.getAppSettings(app or self.__class__.__name__.replace('Handler', '')).get('Value') or {}
+    return self.getSessionData('configuration').get(app or self.__class__.__name__.replace('Handler', ''), {})
+
+  def getTheme(self):
+    """ Get theme
+
+        :return: str
+    """
+    return self.getSessionData('configuration').get("Theme", "tabs")
```

Comment:
Shouldn't it be "Configuration"?

And BTW, didn't we actually say to remove the Desktop theme?

Reply:
IIUC now everywhere reference to "configuration" key name.

Yes, we agree that Desktop must be removed. But, as I remember, after #427 will be solved and not in v4r2, no?